socorro/processor/breakpad_transform_rules.py

```diff
@@ -317,25 +305,27 @@ def _execute_external_process(self, command_line, processor_meta):
 
     def expand_commandline(self, dump_file_pathname, raw_crash_pathname):
         """Expands the command line parameters and returns the final command line"""
-
         # NOTE(willkg): If we ever add new configuration variables, we'll need
         # to add them here, too, otherwise they won't get expanded in the
         # command line.
 
+        symbols_urls = ' '.join([
+            '--symbols-url %s' % url.strip()
```

Comment:
Drive-by-comment; Shouldn't that be `'--symbols-url "%s"'` just in case URL has a character in it that affects the parameter passing.

Reply:
Yes, it should. Thank you!